MonstieWash/Assets/Scripts/Compendium/CompendiumEntry.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class CompendiumEntry : MonoBehaviour
+{
+    public string MonsterName { get => monsterName;}
+    public string Temperament { get => temperament; }
+    public string Description { get => description;}
+    public Sprite ImageOriginal { get => imageOriginal; }
+    public Sprite ImageCleared { get => imageCleared; }
+    public bool Completed { get => completed;}
+
+    [SerializeField] private string monsterName;
+    [SerializeField] [TextAreaAttribute] private string temperament;
+    [SerializeField] [TextAreaAttribute] private string description;
+    [SerializeField] private Sprite imageOriginal;
+    [SerializeField] private Sprite imageCleared;
+    [SerializeField] private bool completed;
+
+    private CompendiumManager m_compendiumManager;
+
+    private void Start()
+    {
+        m_compendiumManager = FindFirstObjectByType<CompendiumManager>();
+    }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        m_compendiumManager.DisplayEntry(this);
+    } 
+}
```

Comment:
This is probably the most significant issue, in that any collider will trigger this function. This means that the arm of the player triggers it as well as the interaction box of the hand, causing some awkward and unintuitive behaviour. 

We want this function to run only when the hand box collider is overlapping. To achieve this, either only check for box colliders as the hand collider is a box2d while the arm collider is a polygon2d, OR mess with the layer interactions of the colliders so that only the hand collides with the monster elements. Up to you how you think is best to handle this, but it'll need to be resolved before it can be merged.